test/language/statements/for-in/head-const-bound-names-in-stmt.js

```diff
@@ -0,0 +1,15 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+description: The body may not re-declare variables declared in the head
+negative: SyntaxError
+info: |
```

Comment:
we've been using the `>` character on the other parts, it's good to use them here for consistency.